Dockerfile

```diff
@@ -236,6 +237,11 @@ ENV DEPENDABOT_NATIVE_HELPERS_PATH="/opt" \
   PATH="$PATH:/opt/terraform/bin:/opt/python/bin:/opt/go_modules/bin:/opt/dep/bin" \
   MIX_HOME="/opt/hex/mix"
 
+# Allow further gem installs as the dependabot user
+ENV BUNDLE_PATH="/home/dependabot/.bundle" \
+    BUNDLE_BIN=".bundle/bin"
+ENV PATH="$BUNDLE_BIN:$PATH:$BUNDLE_PATH/bin"
```

Comment:
When fixing the typo in `bundler/helpers/v1/build` `bundle installs` started failing and this fixed it, I'm pretty sure we want to set these env vars before building the v1 v2 helpers. Not sure why this was working previously.